trieste/acquisition/rule.py

```diff
@@ -444,17 +409,59 @@ def acquire(
 
             is_global = step_is_success or not state.is_global
 
-        if is_global:
-            acquisition_space = search_space
-        else:
-            xmin = dataset.query_points[tf.argmin(dataset.observations)[0], :]
-            acquisition_space = Box(
-                tf.reduce_max([global_lower, xmin - eps], axis=0),
-                tf.reduce_min([global_upper, xmin + eps], axis=0),
-            )
+            if is_global:
+                acquisition_space = self._global_search_space
+            else:
+                xmin = dataset.query_points[tf.argmin(dataset.observations)[0], :]
+                acquisition_space = Box(
+                    tf.reduce_max([global_lower, xmin - eps], axis=0),
+                    tf.reduce_min([global_upper, xmin + eps], axis=0),
+                )
 
-        acquisition_function = self._builder.prepare_acquisition_function(datasets, models)
-        point = self._optimizer(acquisition_space, acquisition_function)
-        state_ = TrustRegion.State(acquisition_space, eps, y_min, is_global)
+            new_state = ContinuousTrustRegionState(acquisition_space, eps, y_min, is_global)
+            return new_state, acquisition_space
+
+        return go
+
+
+def continuous_trust_region(
+    beta: float = 0.7, kappa: float = 1e-4
```

Comment:
I know it's a bad idea but it still makes me sad that we don't write

```suggestion
    β: float = 0.7, κ: float = 1e-4
```

Reply:
i asked this kind of question on a forum for another language recently. They gave a list of reasons why to not do this (which it sounds like you know), but one of them was that unicode is mangled on iOS. Found that mildly interesting